http/get_simple/swift/Client/README.md

```diff
@@ -0,0 +1,45 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# HTTP GET Arrow Data: Simple Swift Client Example
+
```

Comment:
```suggestion

> [!CAUTION]
> This Swift client example is compatible _only_ with the Swift server example. It is incompatible with all the other server examples in the `get_simple` directory of this repository. This is because of a problem in how the Swift Arrow package implements the Arrow IPC stream format (https://github.com/apache/arrow/issues/44910).

```